src/views/postUpdation/TextPostUpdation.tsx

```diff
@@ -45,6 +49,9 @@ export const TextPostUpdation: React.FC<ITextPostUpdationProps> = ({
     post.origins,
   );
   const [htmlContent, setHtmlContent] = useState(post.content);
+  const [previewImageUrl, setPreviewImageUrl] = useState<string>(
```

Comment:
I think it's not needed, typescript should infer type
```suggestion
  const [previewImageUrl, setPreviewImageUrl] = useState(
```

Reply:
Fixed